packages/@aws-cdk/aws-ram/package.json

```diff
@@ -99,7 +99,7 @@
   "engines": {
     "node": ">= 14.15.0"
   },
-  "stability": "experimental",
+  "stability": "stable",
```

Comment:
why? This module only has L1 constructs.

Reply:
I noticed that during build of this new feature in the AppReg constructs that I would run in to an error of these experimental constructs not being able to depend on other experimental constructs (the RAM constructs). Builds would fail because of this. I'm not sure if this is a problem with the RAM constructs or build, but we would like to be able to depend on the RAM constructs for this feature in AppReg.

Reply:
odd, can you give the exact error output? The solution to this is definitely not to make an L1-only library stable

Reply:
From a `yarn build` when `@aws-cdk/aws-ram` stability is `experimental`:
```
In package package.json
- [no-experimental-dependencies] It is not allowed to depend on experimental modules. @aws-cdk/aws-servicecatalogappregistry added a dependency on experimental module @aws-cdk/aws-ram
```
During the build, this error is returned by `pkglint`.

Reply:
PR to unblock: https://github.com/aws/aws-cdk/pull/21208

Reply:
Thank you for that!